### PR TITLE
Nestedlist

### PR DESCRIPTION
**Pull request by SimranLahrani842**
List within a list.
useful-when you want to create a matrix or want a list within list of different data types

### CODE REVIEW

**Comment by SimranLahrani842** [comment]
You can insert ,pop, del the elements of a list within a list using indexes.

File: c.md [hunk 1]
```diff
@@ -471,4 +471,9 @@ All functions defined under `<stdlib.h>` header file. These randomized functions
 
 * `gcc prog.c` : compiles prog.c into a.out, run result with  `./a.out`.
 * `gcc -o prog prog.c` : compiles prog.c into prog, run result with `./prog`.
-* `gcc -g -o prog prog.c` : as above, but allows for debugging.
\ No newline at end of file
+* `gcc -g -o prog prog.c` : as above, but allows for debugging.
+
+##use of fflush()
```
Comment (onecompiler-ops) [inline c.md:476]: Fix markdown issues

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

File: c.md (hunk 1 above)
Comment (SimranLahrani842) [reply c.md]: fixed as per required.

**Review by onecompiler-ops — APPROVED** [review]

**Comment by SimranLahrani842** [comment]
Sir,please add the respected labels of hacktober2022,accepted,etc.

**Comment by onecompiler-ops** [comment]
@SimranLahrani842 added

**Pull request merged**